### ISSUE TITLE
A 67-layer sandwich 

### ISSUE THREAD

**Issue by bryanzk** [SorellaLabs/brontes#818]
### Type of MEV

sandwich

### Etherscan Links to Transactions

The front piece: https://etherscan.io/tx/0xb1bf1bc77ab12f028a34ebef8241028402e035cdcd34bc286063fcd01cf604aa
The back piece:
https://etherscan.io/tx/0xd557586fd97f0961e4a981198e80ad8c1001aa340fd911e906d6a135b924cee1
All the 65 txs in between are victim transactions.

### Protocols Interacted With

Uniswap V2

### All Protocols Supported?

- [X] Yes, all protocols were supported.
- [ ] No, not all protocols were supported.

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

**Comment by Will-Smith11** [comment]
makes sense why we missed it, there is a hard filter cap of 30 vicitims currently, will update. thats pretty wild though

**Comment by 0xvanbeethoven** [comment]
this is nuts
